chromium/.eslintrc.json

```diff
@@ -4,21 +4,21 @@
     "es6": true,
     "webextensions": true
   },
+  "overrides": [
+    {
+      "files": ["background.js", "entry.js", "incognito.js", "rules.js" ,"store.js", "util.js"],
```

Comment:
What's this needed for / why not `module` by default?

Reply:
`module` disables `/* exported */`